lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip-tabbed/search-facet-chip-tabbed.component.scss

```diff
@@ -0,0 +1,13 @@
+.adf-search-filter-chip-tabbed {
```

Comment:
I would suggest to wrap these all styles in component's selector to avoid styles leaking

Reply:
Wrapped